### DISCUSSION TITLE
pythainlp.spell.spell_sent

### DISCUSSION THREAD

**Discussion by PrapakornK** [PyThaiNLP/pythainlp#638] [Q&A] [answered]
according to this https://pythainlp.github.io/dev-docs/api/spell.html , there should be a function that can spell check a list of string rather than one string at a time. I, however, cant use it even though i can use the normal spell function. Is this function removed? and will it come back?

**Comment by wannaphong** [comment, marked as answer]
Can you get examples (words) to me?

**Reply by PrapakornK** [reply]
i was trying the exact example on shown in the link. I havent reached the acutal function yet because i got this when trying to import it.
![image](https://user-images.githubusercontent.com/97608639/149152892-cc4403c1-c143-4910-ae50-6d31f577017e.png)

**Reply by wannaphong** [reply]
from https://pythainlp.github.io/dev-docs/api/spell.html, It's pre-release version of PyThaiNLP. If you install by `pip install pythainlp`, It's PyThaiNLP version 2.3.2. You can read the documentation of PyThaiNLP 2.3 at https://pythainlp.github.io/docs/2.3/api/spell.html.

If you wants use PyThaiNLP pre-release version, you can install by `pip install --pre pythainlp`. Read more [PyThaiNLP v3.0.0-dev0](https://github.com/PyThaiNLP/pythainlp/releases/tag/v3.0.0-dev0).

**Reply by PrapakornK** [reply]
i will try that :D thank you
